parm/config/config.ecfvrfy

```diff
@@ -0,0 +1,198 @@
+        export webhostid="$USER"                        # webhost(rzdm) user name
+        export SEND2WEB="NO"                            # whether or not to send maps to webhost
+        export WEBDIR="/home/people/emc/www/htdocs/gmb/${webhostid}/vsdb/$PSLOT"
+        export mdlist="gfs $PSLOT "                     # exps (up to 10) to compare in maps
+    fi
+fi
+
+
+#----------------------------------------------------------
+# Minimization, Radiance and Ozone Monitoring
+#----------------------------------------------------------
+
+if [ $VRFYRAD = "YES" -o $VRFYMINMON = "YES" -o $VRFYOZN = "YES" ]; then
+
+    # Radiance Monitoring
+    if [[ "$VRFYRAD" == "YES" && "$CDUMP" == "$CDFNL" ]] ; then
+
+        export RADMON_SUFFIX=$PSLOT
+        export TANKverf="$NOSCRUB/monitor/radmon"
+        export VRFYRADSH="$HOMEgfs/jobs/JGDAS_ATMOS_VERFRAD"
+
+    fi
+
+    # Minimization Monitoring
+    if [[ "$VRFYMINMON" = "YES" ]] ; then
+
+        export MINMON_SUFFIX=$PSLOT
+        export M_TANKverf="$NOSCRUB/monitor/minmon"
+        if [[ "$CDUMP" = "gdas" ]] ; then
+            export VRFYMINSH="$HOMEgfs/jobs/JGDAS_ATMOS_VMINMON"
+        elif [[ "$CDUMP" = "gfs" ]] ; then
+            export VRFYMINSH="$HOMEgfs/jobs/JGFS_ATMOS_VMINMON"
+        fi
+
+    fi
+
+    # Ozone Monitoring
+    if [[ "$VRFYOZN" == "YES" && "$CDUMP" == "$CDFNL" ]] ; then
+
+        export HOMEgfs_ozn="$HOMEgfs"
+        export OZNMON_SUFFIX=$PSLOT
+        export TANKverf_ozn="$NOSCRUB/monitor/oznmon"
+        export VRFYOZNSH="$HOMEgfs/jobs/JGDAS_ATMOS_VERFOZN"
+
+    fi
+
+fi
+
+
+#-------------------------------------------------
+# Cyclone genesis and cyclone track verification
+#-------------------------------------------------
+
+export ens_tracker_ver=v1.1.15.5
+if [ $machine = "WCOSS_DELL_P3" ] ; then
+    export ens_tracker_ver=v1.1.15.3
+fi
+export HOMEens_tracker=$BASE_GIT/tracker/ens_tracker.${ens_tracker_ver}
+if [ $machine = "ORION" ] ; then
+    export HOMEens_tracker=$BASE_GIT/tracker/TC_tracker.v1.1.15.2
+fi
+if [ $machine = "WCOSS2" ] ; then
+    export HOMEens_tracker=${BASE_GIT}/TC_tracker/${ens_tracker_ver}
+fi
+if [ "$VRFYTRAK" = "YES" ]; then
+
+    if [ "$CDUMP" = "gdas" ]; then
+        export TRACKERSH="$HOMEgfs/jobs/JGDAS_ATMOS_CYCLONE_TRACKER"
+        export FHOUT_CYCLONE=3            
+        export FHMAX_CYCLONE=$FHMAX
+    else
+        export TRACKERSH="$HOMEgfs/jobs/JGFS_ATMOS_CYCLONE_TRACKER"
+        export FHOUT_CYCLONE=6            
+        export FHMAX_CYCLONE=$(( FHMAX_GFS<240 ? FHMAX_GFS : 240 ))
+    fi
+fi
+
+
+if [[ "$VRFYGENESIS" == "YES" && "$CDUMP" == "gfs" ]]; then
+
+    export GENESISSH="$HOMEgfs/jobs/JGFS_ATMOS_CYCLONE_GENESIS"
+fi
+
+if [[ "$VRFYFSU" == "YES" && "$CDUMP" == "gfs" ]]; then
+
+    export GENESISFSU="$HOMEgfs/jobs/JGFS_ATMOS_FSU_GENESIS"
+fi
+
+if [[ "$RUNMOS" == "YES" && "$CDUMP" == "gfs" ]]; then
+
+    if [ $machine = "WCOSS_C" ] ; then
```

Comment:
Can we move these settings up to the other machine section instead? I also don't know that it matters if we define these only for gfs, if they aren't used anyway. In the above section, I would just either set RUNGFSMOSSH, or set RUNMOS to "NO", based on the machine.

Also question why there is a different script for each machine, but that's outside the scope of this PR.